clients/rge/index.ts

```diff
@@ -0,0 +1,127 @@
+import routes from '#clients/routes';
+import {
+  IRGECompanyCertifications,
+  NomCertificat,
+} from '#models/garant-environement';
+import { Siren, Siret } from '#utils/helpers';
+import { httpGet } from '#utils/network';
+
+/*
```

Comment:
Would add this to js doc to exported client to access it from elsewhere in vscode